packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -29,80 +29,89 @@ function noop( ...args: unknown[] ) {
 	return args;
 }
 
-interface Props {
+interface SummaryStepProps {
+	input: ReactNode;
+	commentary?: ReactNode;
+}
+
+const SummaryStep: React.FunctionComponent< SummaryStepProps > = ( { input, commentary } ) => (
+	<div className="focused-launch-summary__step">
+		<div className="focused-launch-summary__data-input">
+			<div className="focused-launch-summary__section">{ input }</div>
+		</div>
+		<div className="focused-launch-summary__side-commentary">{ commentary }</div>
+	</div>
+);
+
+interface SiteNameStepProps {
+	stepIndex: number;
 	siteId: number;
 }
 
-const Summary: React.FunctionComponent< Props > = ( { siteId } ) => {
+const SiteNameStep: React.FunctionComponent< SiteNameStepProps > = ( { siteId, stepIndex } ) => {
 	const { title, updateTitle, saveTitle } = useTitle( siteId );
+	return (
+		<SummaryStep
+			input={
+				<TextControl
+					className="focused-launch-summary__input"
+					label={
+						<label className="focused-launch-summary__label">
+							{ stepIndex }. { __( 'Name your site', __i18n_text_domain__ ) }
+						</label>
+					}
+					value={ title }
+					onChange={ updateTitle }
+					onBlur={ saveTitle }
+					// eslint-disable-next-line jsx-a11y/no-autofocus
+					autoFocus={ true }
+				/>
+			}
+		/>
+	);
+};
 
+interface DomainStepProps {
+	stepIndex: number;
+}
+
+const DomainStep: React.FunctionComponent< DomainStepProps > = ( { stepIndex } ) => {
 	const site = useSite();
 
 	const [ siteDomainName ] = React.useState( site.currentDomainName );
 	const domainSearch = useDomainSearch();
 
 	return (
-		<div className="focused-launch-summary__container">
-			<div className="focused-launch-summary__section">
-				<Title>{ __( "You're almost there", __i18n_text_domain__ ) }</Title>
-				<p className="focused-launch-summary__caption">
-					{ __(
-						'Prepare for launch! Confirm a few final things before you take it live.',
-						__i18n_text_domain__
-					) }
-				</p>
-			</div>
-			<div className="focused-launch-summary__step">
-				<div className="focused-launch-summary__data-input">
-					<div className="focused-launch-summary__section">
-						<TextControl
-							className="focused-launch-summary__input"
-							label={
+		<SummaryStep
+			input={
+				<>
+					<DomainPicker
+						header={
+							<>
 								<label className="focused-launch-summary__label">
-									{ __( '1. Name your site', __i18n_text_domain__ ) }
+									{ stepIndex }. { __( 'Confirm your domain', __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( '2. Confirm your domain' )` *ES Score: 9*